src/components/TreePage.vue

```diff
@@ -56,11 +68,10 @@ li {
 a {
   color: #42b983;
 }
-.tree {
+#photo {
```

Comment:
NIT: can we use class names?

Reply:
I use `getElementById` to get a handle to the element and load the (background) image, although may be it can be done using a `ref` and `this.$refs.photo`.